### PR TITLE
Lesson33 ezgi

### PR DESCRIPTION
**Pull request by EZGIAKYLDZ**

### CODE REVIEW

File: Lesson33/Ezgi/lesson33-app/src/Exercise10/index.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    function doubleClickHandler () {
+        setClickCount(clickCount + 1);
+
+        if(clickCount === 2) {
+            alert('Button was double-clicked!');
+            setClickCount(0);
+        }
+    }
+    return (
+        <button onClick={doubleClickHandler}>Double click me</button>
```
Comment (annasvst) [inline Lesson33/Ezgi/lesson33-app/src/Exercise10/index.jsx:16]: Here you can use 'onDoubleClick' event listener instead

File: Lesson33/Ezgi/lesson33-app/src/Exercise8/index.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    const [listItems, setItems] = useState(
+        [
+            "Finish homeworks",
+            "Go to Market",
+            "Gym after work",
+            "Meeting Friends"
+        ]
+    )
+
+    const deletekHandler = (removeToIndex) => {
```
Comment (annasvst) [inline Lesson33/Ezgi/lesson33-app/src/Exercise8/index.jsx:13]:
```suggestion
    const deleteHandler = (indexToRemove) => {
```

File: Lesson33/Ezgi/lesson33-app/src/Exercise8/index.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+       setItems(newList);
+    }
+
+
+    return (
+        <ul>
+            {listItems.map((item, index) => (
+                <li key={index}>
+                    {item} 
+                    <button onClick={() => deletekHandler(index)}>Delete</button>
```
Comment (annasvst) [inline Lesson33/Ezgi/lesson33-app/src/Exercise8/index.jsx:24]:
```suggestion
                    <button onClick={() => deleteHandler(index)}>Delete</button>
```

File: Lesson33/Ezgi/lesson33-app/src/Exercise8/index.jsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        [
+            "Finish homeworks",
+            "Go to Market",
+            "Gym after work",
+            "Meeting Friends"
+        ]
+    )
+
+    const deletekHandler = (removeToIndex) => {
+       const newList =listItems.filter((item,index) => index !== removeToIndex)
```
Comment (annasvst) [inline Lesson33/Ezgi/lesson33-app/src/Exercise8/index.jsx:14]:
```suggestion
       const newList =listItems.filter((item,index) => index !== indexToRemove) 
```

File: Lesson33/Ezgi/lesson33-app/src/Exercise10/index.jsx (hunk 1 above)
Comment (EZGIAKYLDZ) [reply Lesson33/Ezgi/lesson33-app/src/Exercise10/index.jsx:16]: I hadn't thought of that. I've changed it to onDoubleClick

**Review by annasvst — APPROVED** [review]

**Pull request merged**